synthetic_data/generators.py

```diff
@@ -1,20 +1,25 @@
 """Contains generators for tabular, graph, and unstructured data profiles."""
 
 import dataprofiler as dp
+import numpy as np
 from sklearn import preprocessing
 
 from synthetic_data.base_generator import BaseGenerator
+from synthetic_data.dataset_generator import generate_dataset
 from synthetic_data.graph_synthetic_data import GraphDataGenerator
 from synthetic_data.synthetic_data import make_data_from_report
 
 
 class TabularGenerator(BaseGenerator):
     """Class for generating synthetic tabular data."""
 
-    def __init__(self, profile, seed=None, noise_level: float = 0.0):
+    def __init__(
+        self, profile, seed=None, noise_level: float = 0.0, method: str = "correlated"
```

Comment:
wonder if `method` should be a boolean instead of a str option... like we could do `correlated: bool = True` and then limit to two option of correlated and simple...